cdn-uploader.js

```diff
@@ -0,0 +1,41 @@
+const ftp = require("basic-ftp");
+
+const config = {
+	host: process.env.AKAMAI_CDN_HOSTNAME,
+	user: process.env.AKAMAI_CDN_USERNAME,
+	password: process.env.AKAMAI_CDN_PASSWORD,
+	cpCode: process.env.AKAMAI_CDN_CPCODE,
+	port: 21,
+}
+
+function UploadFilesFromPath() {
+	console.log("[+] Connecting to CDN...");
+	async function Upload() {
+		try {
+			const client = new ftp.Client();
+			await client.connect(config.host, config.port);
+			await client.login(config.user, config.password);
+			await client.useDefaultSettings();
+			console.log(`[+] Starting to upload the content of "${config.buildPath}".`);
+			await client.ensureDir(`/${config.cpCode}/${config.basePath}/${config.buildPath}`);
+			await client.clearWorkingDir();
+			await client.uploadDir(config.buildPath);
+			console.log(`[+] Folder "${config.buildPath}" uploaded.`);
+			client.close();
+		}
+		catch(err) {
+			console.log(err);
+		}
+	}
+	Upload();
+}
+
+function GetFoldersArguments(){
+	var args = [].slice.call(process.argv);
+	args.splice(0, 2);
+	config.basePath = args[0];
+	config.buildPath = args[1];
+}
+
+GetFoldersArguments();
+UploadFilesFromPath();
```

Comment:
Very minor, but what do you think about something like this to avoid access to global variables from the functions:

```js
const ftp = require("basic-ftp");
const config = { . . . }
function UploadFiles(config) { . . . }
function UpdateConfigWithArgs(argv, config){
	var args = [].slice.call(argv);
	args.splice(0, 2);
	config.basePath = args[0];
	config.buildPath = args[1];
}
UpdateConfigWithArgs(process.argv, config);
UploadFiles(config);
```